client/components/happychat/button.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import viewport from 'lib/viewport';
+import { openChat } from 'state/ui/happychat/actions';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+
+const HappychatButton = ( { translate, onOpenChat } ) => (
```

Comment:
Just a comment here. The component doesn't need be wrapped into parenthesis.

```es6
const HappychatButton = ( { translate, onOpenChat } ) =>
	<Button compact borderless onClick={ onOpenChat }>
		<Gridicon icon="comment" /> { translate( 'Support Chat' ) }
	</Button>;
```

Please ignore this comment if you think to use parentheses is more clear and visually better.